package.json

```diff
@@ -74,5 +75,7 @@
     "react-dom": "^16.3.2",
     "rimraf": "^2.6.2"
   },
-  "peerDependencies": {}
+  "peerDependencies": {
+    "babel-core": "^6.0.0 || ^7.0.0-0"
```

Comment:
the peer dep range should match the dep range; otherwise someone who installs babel-core 7 will still have this project using v6.

Reply:
Does that effectively mean we are telling people you can't use babel 7 with this package?

Reply:
Yes - if you want to support both babel 6 and babel 7, then it needs to only be a peer dep, which means the user has to explicitly choose one.

Reply:
Hmm, but unfortunately in that case, if the user selected babel 7, then `babel-loader` would complain about a missing peer-dependency since it's set to `babel-core@6` based on the warning, and it seem loader 8 only support babel 7 and not 6.

@trotzig I think this is a product decision for you.

Reply:
Interesting, it seems yarn doesn't handle these correctly. After making these dependencies dual, a `yarn install` is still warning me about not having the correct peers. 

I think I'm going to revert back to using regular dependencies for now and revisit if we find it to be a problem in the wild. For most current happo users, the babel dependencies overlap with what's in their own package.json which means babel-core, webpack, etc will get hoisted to the root of node_modules.

Reply:
That's fine, but really that's just a flaw in yarn - the consequence of doing "just deps" is that someone can be using babel 7 in their code, but happo will be using babel 6, and they'll get different - possibly unexpected - results.

One possibility I'd recommend is to abandon yarn and use the ecosystem's standard package manager, npm.